services/workflows-service/src/env.ts

```diff
@@ -95,6 +95,13 @@ export const serverEnvSchema = {
   IN_MEMORIES_SECRET_ACQUIRER_ID: z.string().optional(),
   IN_MEMORIES_SECRET_PRIVATE_KEY: z.string().optional(),
   IN_MEMORIES_SECRET_CONSUMER_KEY: z.string().optional(),
+  TELEMETRY_ENABLED: z
```

Comment:
Can we consider renaming this variable? With OTEL telemetry also in use, the current name might cause confusion."
`ANALYTICS_ENABLED`
or
`SUPABASE_TELEMETRY_ENABLED`